src/main/kotlin/uk/gov/dwp/dataworks/logging/LogFields.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.dwp.dataworks.logging
+
+import java.net.InetAddress
+
+/**
+ * Object to hold the Application specific and Common fields that should be added to each of the log lines output by
+ * this library. Provides helper methods to return these fields as a set of json compliant Key:value pairs but without
+ * any wrapping (i.e no `{}`).
+ */
+object LogFields {
+    private val logFields = CommonLogFields.commonFields.toMutableMap()
+    val asJson = logFields.map { """ "${it.key}":"${it.value}" """.trim() }.joinToString(separator = ",")

```

Comment:
why add whitespace and them trim?

Reply:
I did it for code readability. Now that you point it out this will be a performance overhead. I will remove this and other instances of the pattern